rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -401,3 +401,30 @@ dfs profile for search with dfs_query_then_fetch:
 
   - match: { hits.total.value: 0 }
   - is_false: profile.shards.0.dfs
+---
+composite_id_parsed:
+  - skip:
+      version: " - 8.8.99"
```

Comment:
With 8.9.0 being cut, you will need to update this as it will get released in 8.10 now.

Reply:
Fixed. Thanks for the reminder.

Reply:
I suppose you haven't pushed your local commit yet? Once you do, I will give it another review

Reply:
Pushed it now. It must have failed when I tried it earlier and I missed it.